Client/src/Pages/Incidents/IncidentTable/index.jsx

```diff
@@ -196,7 +197,9 @@ const IncidentTable = ({ monitors, selectedMonitor, filter }) => {
 												/>
 											</TableCell>
 											<TableCell>{formattedDate}</TableCell>
-											<TableCell>{check.statusCode ? check.statusCode : "N/A"}</TableCell>
+											<TableCell>
+												<HttpStatusLabel status={check.statusCode || "N/A"} />
```

Comment:
The implementation is great. Something I was thinking, what if status code is somthing other than undefined, and it isn't a number between 100 and 600? We sould be sending an uncovered scenario to the HttpStatusLabel component. 

What I suggest is we just pass down chueck.statusCode, and inside of http status label we deal with that

Reply:
Agreed, let's handle the logic inside of `HttpStatusLabel` since we're dealing with it there for colors anyway